seqr/views/utils/test_utils.py

```diff
@@ -1238,7 +1238,7 @@ def call_request_json(self, index=-1):
                     'gerp_rs': '5.07', 'haplogroup_defining': None, 'metasvm': None, 'mitotip': None,
                     'mpc': None, 'mut_taster': 'N', 'phastcons_100_vert': '0.958000', 'polyphen': None,
                     'primate_ai': None, 'revel': None, 'sift': 'D', 'splice_ai': None, 'splice_ai_consequence': None,
-                    'strvctvre': None},
+                    'strvctvre': None,'gnomad_noncoding': None,},
```

Comment:
You may want to add a blank before `'gnomad_noncoding'`.

Reply:
no, if you look at all the line above it you will see that these are not each on their own line